code/modules/crafting/anvil_recipes/weapons.dm

```diff
@@ -421,6 +421,22 @@
 	created_item = /obj/item/weapon/sword/long/greatsword/zwei
 	craftdiff = 3
 
+/datum/anvil_recipe/weapons/iron/flamberge
+	name = "Flamberge (+Bar x2)"
+	recipe_name = "a Flamberge"
+	appro_skill = /datum/skill/craft/weaponsmithing
+	additional_items = list(/obj/item/ingot/iron, /obj/item/ingot/iron)
+	created_item = /obj/item/weapon/sword/long/greatsword/flamberge
+	craftdiff = 3
+
```

Comment:
For the same reason that /obj/item/weapon/sword/long/rider is not craftable and other mercs/rare_adv weapons, those are rare adventurer/migrant_wave/mercs weapons, they are to be treated as unique and exotic, making it craftable just ruins the flavour of these roles